src/org/zaproxy/zap/extension/invoke/resources/help/contents/options.html

```diff
@@ -54,6 +54,16 @@ <H3>Parameters</H3>
 			<td>%postdata%</td>
 			<td>the POST data sent, if any, with any newlines replaced with "\n"</td>
 		</tr>
+		<tr>
+			<td>&nbsp;&nbsp;&nbsp;&nbsp;</td>
+			<td>%msgid%</td>
+			<td>the HTTP message id to fetch data the API (/JSON/core/view/message/?zapapiformat=JSON&id=$msgid)"\n"</td>
+		</tr>
+		<tr>
+			<td>&nbsp;&nbsp;&nbsp;&nbsp;</td>
+			<td>%header-{{ header }}%</td>
+			<td>the request header by name (if any),  e.g. %header-user-agent% would return User-Agent header "\n"</td>

```

Comment:
- `<code>%header-user-agent%</code> would return the value of User-Agent header`?
 - `"\n"` should be removed.